datasets/random_dataset.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
```

Comment:
put this file into `conftest.py` under `tests` folder and use decorator: `@pytest.fixture`. This is used specifically in py_test.